webapp/channels/src/components/advanced_text_editor/use_post_box_indicator.test.tsx

```diff
@@ -96,6 +96,7 @@ describe('useTimePostBoxIndicator', () => {
         expect(current.showDndWarning).toBe(false);
         expect(current.isSelfDM).toBe(false);
         expect(current.isBot).toBe(true);
+        expect(current.showRemoteUserHour).toBe(false);
```

Comment:
Could you also add the expected values for these to the other cases as well? That way, we can compare that the returned value is false for bots and true for non-bots